flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendTools.java

```diff
@@ -1107,7 +1107,7 @@ private String buildBadVersionString(String tool, String version,
             String... extraUpdateInstructions) {
         StringBuilder extraInstructions = new StringBuilder();
         for (String instruction : extraUpdateInstructions) {
-            extraInstructions.append("%n  - or ").append(instruction);
+            extraInstructions.append("\n  - or ").append(instruction);
```

Comment:
What about using `System.lineSeparator` instead of hard-coded `\n`, so it should work as `%n` in `String.format()`?

```suggestion
            extraInstructions.append(System.lineSeparator()).append("  - or ")
                    .append(instruction);
```